### ISSUE TITLE
NVDA doesn't read notifications of Windows 10 version 10.0.17134523

### ISSUE THREAD

**Issue by dreinn** [nvaccess/nvda#9242]
With NVDA 2018.4 or latest alpha of February 4 2019, NVDA doesn't announce notifications in windows10.
I installed also the windows10 Essentials addon, but nothing is changed.

Do you confirm this issue?

**Comment by Adriani90** [comment]
cc: @josephsl

**Comment by josephsl** [comment]
Hi, I need a debug log with Windows 10 App Essentials running (restart with debug logging enabled, perform a task that causes toasts to appear, and copy and paste that fragment here). Thanks.

**Comment by dreinn** [comment]
I'm really sorry, it was a my mistake.
The checkbox "report notifications" on NVDA's preferences was disabled.
Consider the ticket closed, and sorry for this.
